### PR TITLE
Update Logic class diagram

### LINKED ISSUE #153: Update class diagram of Logic component
Add association from `LogicManager` to `Storage`

### PR DESCRIPTION
**Pull request by w2vgd**
Closes #153 

Updated logic diagram:
image

Sequence diagram for add property:
image

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -62,11 +62,11 @@ The sections below give more details of each component.
 ![Structure of the UI Component](images/UiClassDiagram.png)
 
 **API** :
-[`Ui.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/ui/Ui.java)
+[`Ui.java`](https://github.com/AY2021S2-CS2103T-T13-4/tp/blob/master/src/main/java/seedu/address/ui/Ui.java)
 
 The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class.
```
Comment (candyhy) [inline docs/DeveloperGuide.md]: Should `PersonListPanel` be updated to `PropertyListPanel` and `AppointmentListPanel`?

**Review by candyhy — COMMENTED** [review]
LGTM, just a small documentation issue

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (w2vgd) [reply docs/DeveloperGuide.md]: Thanks, merged and updated this branch

**Review by candyhy — APPROVED** [review]
LGTM

**Pull request merged**